lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -23,6 +23,7 @@
 short_description: Add or remove YUM repositories
 description:
   - Add or remove YUM repositories in RPM-based Linux distributions.
+  - If you wish to update an existing repository definition use ini_file instead
```

Comment:
The `ini_file` should be decorated like this:

```M(ini_file)```

and there is dot missing at the end of the line.

Reply:
Done enw commit to fix those